examples/five_by_five.py

```diff
@@ -1,24 +1,21 @@
 """Simple version of 5x5, developed for/with Textual."""
 
 from pathlib import Path
-from typing import cast
-import sys
+from typing_extensions import TYPE_CHECKING, cast
```

Comment:
Testing this with Python 3.11.1, attempting to import `cast` from `typing_extensions` gives me this:

```
Traceback (most recent call last):
  File "/Users/davep/temp/textual/examples/five_by_five.py", line 4, in <module>
    from typing_extensions import TYPE_CHECKING, cast
ImportError: cannot import name 'cast' from 'typing_extensions' (/Users/davep/temp/textual/.venv/lib/python3.11/site-packages/typing_extensions.py)
```